### PR TITLE
Planned priority is now 365 calendar days

### PR DESCRIPTION
**Pull request by siapankina**
Planned priority is now calculated as 365 calendar days 
+ updated warning text as design requested

Screenshot 2022-02-03 at 14 03 34

### CODE REVIEW

File: src/utils/hact/workOrderSchedule/raiseWorkOrderForm.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ export const buildScheduleWorkOrderFormData = (workOrderData) => {
         lowPriority: !HIGH_PRIORITY_CODES.includes(
           Number.parseInt(workOrderData.priorityCode)
         ),
+        plannedPriority: Number.parseInt(workOrderData.priorityCode) == 9,
```
Comment (NeilvB) [inline src/utils/hact/workOrderSchedule/raiseWorkOrderForm.js:22]: Use the constant we've defined in `priorities.js` for this, with a `===`

File: src/utils/helpers/completionDateTimes.js [hunk 2]
```diff
@@ -78,12 +78,15 @@ export const calculateCompletionDateTime = ({
   workingDays,
   workingHours = 0,
   lowPriority = false,
+  plannedPriority = false,
```
Comment (NeilvB) [inline src/utils/helpers/completionDateTimes.js:81]: Please write a Jest test for this

**Review by NeilvB — CHANGES_REQUESTED** [review]

File: src/utils/hact/workOrderSchedule/raiseWorkOrderForm.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ export const buildScheduleWorkOrderFormData = (workOrderData) => {
         lowPriority: !HIGH_PRIORITY_CODES.includes(
           Number.parseInt(workOrderData.priorityCode)
         ),
-        plannedPriority: Number.parseInt(workOrderData.priorityCode) == 9,
+        plannedPriority: Number.parseInt(workOrderData.priorityCode) === 9,
```
Comment (NeilvB) [inline src/utils/hact/workOrderSchedule/raiseWorkOrderForm.js:22]: We need to use `PLANNED_PRIORITY_CODE` here

File: src/utils/hact/workOrderSchedule/raiseWorkOrderForm.test.js [hunk 4]
```diff
@@ -162,4 +188,95 @@ describe('buildScheduleWorkOrderFormData', () => {
     const response = buildScheduleWorkOrderFormData(workOrderData)
     expect(response).toEqual(scheduleWorkOrderFormData)
   })
+
+  it('builds the ScheduleWorkOrder form data to post to the Repairs API with Planned Priority', async () => {
```
Comment (NeilvB) [inline src/utils/hact/workOrderSchedule/raiseWorkOrderForm.test.js:192]: 👍  This is a good test to add, which I didn't think of. However we will still benefit from a test at the unit level in this file `completionDateTimes.test.js`. Sorry - I wasn't clear enough in my earlier comment.

You can just add another `describe` at the top level e.g. `'when the plannedPriority is true'`. I think it's important that these target times are put through their paces so having the additional coverage should be worth it.

**Review by NeilvB — CHANGES_REQUESTED** [review]

File: src/utils/hact/workOrderSchedule/raiseWorkOrderForm.js (hunk 3 above)
Comment (siapankina) [reply src/utils/hact/workOrderSchedule/raiseWorkOrderForm.js:22]: ah, sorry for that

**Review by pixeltrix — APPROVED** [review]

**Pull request merged**